documentation/src/main/asciidoc/topics/proc_cli_xsite_push.adoc

```diff
@@ -0,0 +1,13 @@
+[id='cli_xsite_push-{context}']
```

Comment:
@tristantarrant hi, this PR adds two topics to CLI docs for xsite with examples for bringing sites online/offline and state transfer. could you please take a look? thx.